pkg/types/apis/types.go

```diff
@@ -78,3 +78,11 @@ type UpgradeInput struct {
 	PackageName   string `json:"package-name,omitempty"`
 	PackagePath   string `json:"package-path,omitempty"`
 }
+
+type Addon struct {
+	Name        string   `json:"name" wrangler:"required,noupdate"`
+	Description string   `json:"description,omitempty"`
+	FromFile    string   `json:"from-file,omitempty"`
+	Manifest    string   `json:"manifest,omitempty"`
+	Values      []string `json:"values,omitempty"`
```

Comment:
If we don't need to store values for Addons, it might be better using type `map[string]string` as it can prevent duplicated keys.

Reply:
Done